product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx

```diff
@@ -29,16 +29,21 @@ The procedures and functions available in the `UTL_FILE` package are listed in t
 | `FCLOSE_ALL`                                                                   | n/a         | Closes all open files.                                                                                                                                                           |
 | `FCOPY(location, filename, dest_dir, dest_file [, start_line [, end_line ] ])` | n/a         | Copies `filename` in the directory identified by `location` to file `dest_file` in directory `dest_dir`, from line `start_line` to line `end_line`.               |
 | `FFLUSH(file)`                                                                 | n/a         | Forces data in the buffer to be written to disk in the file identified by `file`.                                                                                                |
+| `FGETATTR(location, filename, exists, file_length, block_size)`                | n/a         | Retrieves the attributes for a file when given its `filename` and `location`.                                                                                                    |
+| `FGETPOS(file)`                                                                | `INTEGER`   | Returns an integer which is the relative offset position within the opened file in bytes.                                                                                                                             |
```

Comment:
```suggestion
| `FGETPOS(file)`                                                                | `INTEGER`   | Returns an integer, the relative offset position within the opened file in bytes.                                                                                                                             |
```

Reply:
Returns the relative offset position in the opened file as an integer, in bytes.